R/palaeorotate.R

```diff
@@ -423,43 +362,34 @@ palaeorotate <- function(occdf, lng = "lng", lat = "lat", age = "age",
                    "connected to the internet."),
              call. = FALSE)
       })
-
-    # Set-up for point method -------
     # Define root
     pbase <- "https://gws.gplates.org/reconstruct/reconstruct_points/"
     # Setup query structure
     query <- list(time = NULL, lons = NULL, lats = NULL,
                   model = NULL, anchor_plate_id = 0,
                   return_null_points = TRUE)
-    query <- list(MULLER2022 = query,
-                  MERDITH2021 = query,
-                  MULLER2019 = query,
-                  MULLER2016 = query,
+    query <- list(MERDITH2021 = query,
                   PALEOMAP = query,
+                  TorsvikCocks2017 = query,
                   MATTHEWS2016_pmag_ref = query,
-                  MATTHEWS2016_mantle_ref = query,
-                  GOLONKA = query,
-                  SETON2012 = query)
+                  GOLONKA = query)
     # Define GPlates model names
-    query[["MULLER2022"]]$model <- "MULLER2022"
     query[["MERDITH2021"]]$model <- "MERDITH2021"
-    query[["MULLER2019"]]$model <- "MULLER2019"
-    query[["MULLER2016"]]$model <- "MULLER2016"
     query[["PALEOMAP"]]$model <- "PALEOMAP"
+    query[["TorsvikCocks2017"]]$model <- "TorsvikCocks2017"
+    # anchor plate id 1 required for palaeomagnetic reference frame
+    query$TorsvikCocks2017$anchor_plate_id <- 1
```

Comment:
Not required for MATTHEWS2016_pmag_ref?

Reply:
Oddly, MATTHEWS2016_pmag_ref and MATTHEWS2016_mantle_ref are implemented as two separate endpoints, and the anchor_plate_id does not need defining based on the [GWS docs](https://gwsdoc.gplates.org/models#matthews2016_pmag_ref).